service/src/main/resources/HTMLTemplate.html

```diff
@@ -521,11 +521,12 @@ <h3 class="heading heading--two">Child
     <div class="data-list__row">
       <dt class="data-list__title">Child's living situation</dt>
       <dd class="data-list__definition">
-        {% if case_data.children is not empty and
-        case_data.children.firstChild is not empty and
-        case_data.children.firstChild.livingSituation is not empty %}
+        {% if case_data.children is not empty and case_data.children.firstChild is not empty and case_data.children.firstChild.livingSituation is not empty %}
```

Comment:
I like that one line format better, easier to read IMO despite longer line.

Reply:
same, I find it very confusing to have these multiline if statements